test/targets/node/fetch.js

```diff
@@ -51,6 +51,34 @@ let options = {
   body: '{"number":1,"string":"f\\"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}'
 };
 
+fetch(url, options)
+  .then(res => res.json())
+  .then(json => console.log(json))
+  .catch(err => console.error('error:' + err));`)
+  })
```

Comment:
left in debug?

Reply:
that's part of the output we're comparing to. it's in the application-json request and output fixtures.